doc/api/modules.md

```diff
@@ -232,12 +232,15 @@ RESOLVE_BARE_SPECIFIER(DIR, X)
 2. If X matches this pattern and DIR/name/package.json is a file:
    a. Parse DIR/name/package.json, and look for "exports" field.
    b. If "exports" is null or undefined, GOTO 3.
-   c. Find the longest key in "exports" that the subpath starts with.
-   d. If no such key can be found, throw "not found".
-   e. let RESOLVED_URL =
+   c. If "exports" is a string, or object whose first key does not start with
```

Comment:
@guybedford based on our conversation, did we want to explore removing the string sugar to keep the design space open? or can/should that be done in a PR after this lands?

Reply:
Yes, I was planning to bring up the sugaring questions again once we merge as a separate follow-up, but with this merged first.

Reply:
Marking as resolved for now to follow up separately.